ref/general/microshed.adoc

```diff
@@ -61,30 +59,27 @@ The @Container annotation injects a REST _Client_ proxy of the PersonService, wh
     public static PersonService personSvc;
 
     @Test
-    public void testGetPerson() {
+    public void testGetPerson()
 
 The @RESTClient annotation invokes an HTTP POST request to the running container, which triggers the PersonService#createPerson endpoint and returns the generated ID
         Long bobId = personSvc.createPerson("Bob", 24);
 
-  Using the generated ID, invoke an HTTP GET request to read the record we just created. The JSON response will be automatically converted to a 'Person' object using JSON-B
+By using the generated ID, invoke an HTTP GET request to read the record created. The JSON response automatically converts to a 'Person' object using JSON-B.
```

Comment:
suggested edit- By using the generated ID, invoke an HTTP GET request to read the record that you created. The JSON response automatically converts to a 'Person' object by using JSON-B.

I think you need something between record and created- otherwise it is a passive construction (ie  it really says "record _that was_ created")